content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -0,0 +1,175 @@
+
+The main focus of the web app is to demonstrate how to instrument an application,
+no matter what programming language, platform or OS it uses. The web app also
+shows different instrumentation techniques: automatic and manual, metrics,
+and baggage. All while following the standards and conventions proscribed
+in the official OTel documentation. More about the specific requirements can be
+[found here](https://github.com/open-telemetry/opentelemetry-demo/tree/main/docs/requirements).
+
+My company was focused on becoming part of and embracing the OpenTelemetry
+community. One of our goals this summer was to get more involved with a core
+OpenTelemetry project where we could provide a meaningful contribution.
+The OTel demo was the best match for our goal, as contributing would not
+only help the community, but also provide a great example to test
+and showcase our product with.
+
+The first thing we did was to get in contact with Carter Socha, the organizer of
+the OTel Demo SIG. Carter was really welcoming and helped us identify where our
+contributions could be the most impactful. We started looking at
+[the issue created by Austin Parker](https://github.com/open-telemetry/opentelemetry-demo/issues/39)
+referencing a complete front-end overhaul that would involve moving the
+application away from Go (Server Side Render) to an architecture that included a browser-side
+client (Client Side Render), as well as improving the overall style, theme, and user
+experience.
+
+A fun aspect of the work was the request to move the store from a "normal" store
+to an astronomy store to match the OpenTelemetry project’s overall branding.
+
+Once we got the green light from the rest of the OTel demo SIG,
+then we started working on the different changes that were part
+of the front end architecture overhaul.
+
+## OpenTelemetry Demo Application Description and Tech Stack
+
+The demo app is an astronomy store, with basic eCommerce features such as a
+shopping cart, currency selector, payment, and checkout. It also includes
+the ability to display promotions (ads) and related products depending on
+the user’s context.
+
+The demo’s stack includes multiple microservices in different languages,
+covering each of the following OTel-supported languages:
+
+1. Go
+2. PHP
+3. C++
+4. .NET
+5. Ruby
+6. Python
+7. Node.js
+8. Rust
+9. Elixir
+
+Every microservice has a specific goal and can communicate with others by using
+a global GRPC definition. Persistent information is saved into a Postgres
+database and there are outbound services that connect with third-party services
+to trigger events (such as confirmation emails). All of the microservices,
+including the front end, are connected to the same Open Telemetry collector
+instance, which uses Jaeger as one of the data stores for the traces and spans.
+
+![Open Telemetry Demo System Diagram](system-diagram.png)
+
+Prior to re-architecting, the front end consisted of a Golang Server Side Rendered app,
```

Comment:
Why uppercase "Server Side Rendered"?

Reply:
Since it's been explained earlier in the post, the first time the acronym was used (I think it's line 33), you can use "SSR" here onwards.